bin/pyrdp-mitm.py

```diff
@@ -41,7 +43,14 @@ def main():
     reactor.adoptStreamPort(s.fileno(), socket.AF_INET, MITMServerFactory(config))
     s.close()  # reactor creates a copy of the fd.
 
-    logger.info("MITM Server listening on port %(port)d", {"port": config.listenPort})
+    message = "MITM Server listening on port %(port)d"
```

Comment:
Stating the bind-address in the logs is more accurate and might prevent surprises from users since HOST_IP != how we bind:

```suggestion
    message = "MITM Server listening on 0.0.0.0:%(port)d"
```